lib/config/config-validator.js

```diff
@@ -97,7 +101,10 @@ function validateRuleOptions(id, options, source) {
 
         if (!validSeverity) {
             message.push(
-                "\tSeverity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed \"", severity, "\").\n");
+                "\tSeverity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed `",
+                util.inspect(severity).replace(/'/g, "\"").replace(/\n/g, ""),
```

Comment:
The reason why I introduced [`util.inspect`](https://nodejs.org/api/util.html#util_util_inspect_object_options) is:

```js
String(['error']); //=> error <- no brackets
JSON.stringify(['error']); //=> ["error"]
util.inspect(['error']); //=> [ 'error' ]
```

```js
String(function fn() {}); //=> function fn() {}
JSON.stringify(function fn() {}); //=> undefined
util.inspect(function fn() {}); //=> [Function: fn]


```